authz/audit/stdout_logger.go

```diff
@@ -0,0 +1,64 @@
+/*
+ *
+ * Copyright 2023 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package audit
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+type StdOutLogger struct{}
+
+func (logger *StdOutLogger) Log(event *Event) {
+	jsonBytes, err := json.Marshal(event)
+	if err != nil {
+		fmt.Errorf("failed to marshal log data to JSON: %v", err)
+	}
+
+	fmt.Println(string(jsonBytes))
```

Comment:
Use the `log` package instead of `fmt.Println`

Reply:
+1. Thread-safety is needed and `log` should be able to support that.

Reply:
embedded grpc logger `var grpcLogger = grpclog.Component("name")' requires env variables to be set (like `GRPC_GO_LOG_SEVERITY_LEVEL=info`) but it might not work for all the users. Do you suggest to use another Go logger for it?

Reply:
switched to log.go package